vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_demos/config_files/FreeRTOSIPConfig.h

```diff
@@ -242,7 +242,7 @@ extern uint32_t numaker_ulRand(void);
 #define ipconfigUSE_TCP                                ( 1 )
 
 /* USE_WIN: Let TCP use windowing mechanism. */
-#define ipconfigUSE_TCP_WIN                            ( 0 )
+#define ipconfigUSE_TCP_WIN                            ( 1 )
```

Comment:
How does the build pass for other boards (ST, TI, NXP ) that also do not have this macro set?

Reply:
I looked at the call-sites for `ulTCPWindowTxSack`, and I see only a single call from [`prvReadSackOption`](https://github.com/aws/amazon-freertos/blob/master/libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_TCP_IP.c#L1336) that is currently protected by [`ipconfigUSE_TCP_WIN`](https://github.com/aws/amazon-freertos/blob/master/libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_TCP_IP.c#L1302-L1322) 
Does the problem not exist anymore?

Reply:
It seems that `prvSkipPastRemainingOptions` was removed in #2015 by @htibosch

Reply:
Yeah, I will rebase to the latest master and check if Nuvoton eth project can build.